models/RequestModel.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const RequestModel = new Schema(
+  {
+    act: { type: String, required: false },
+    name: { type: String, required: true },
+    mobile: { type: String, require: true },
+
+    address: { type: String, required: false },
+    area: { type: String, required: true },
```

Comment:
should area be required, in front end forms it's kept as optional

Reply:
In "Request for Help", I guess, we need to know in which area the help is required. That's why I made it as mandatory.